app/decorators/case_contact_decorator.rb

```diff
@@ -47,7 +47,7 @@ def contact_types
   end
 
   def report_contact_types
-    object.contact_types&.map { |ct| ct.humanize.titleize }.join("|")
+    object.contact_types&.map { |ct| ct.humanize.titleize }&.join("|")
```

Comment:
This should never be null in a saved case contact, I want to fix the underlying issue if this is causing issues

Reply:
The issue was that we are doing the `&.map` but then calling join (on something that supposedly might not exist. Should we take both `&` out instead?

Reply:
OMG you are so quick 😂